sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1625,7 +1420,13 @@ def task_worker(self):
                             # Apply module SI settings if applicable
                             lane_speed = int(speed/1000)//host_lane_count
                             optics_si_dict = optics_si_parser.fetch_optics_si_setting(pport, lane_speed, sfp)
-
+                            
+                            helper_logger.log_debug("Read SI parameters for port {} from optics_si_settings.json vendor file:".format(lport))
```

Comment:
Please define log_debug (similar to log_error and log_notice) in CmisManagerTask and use it in CMIS related functions.